activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -119,7 +134,7 @@ class Array
     alias :orig_sum :sum
```

Comment:
If Array#sum is no longer Numeric-only, we'll need to update the comment and `!(%w[a].sum rescue false)` feature detection too.